<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MLP Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background: #202020;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="networkTypeLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	border-style: solid;
	border-width: 4px 0px 4px 0px;
	border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #30D5C8 , stop:1 #4169E1);
}</string>
   </property>
   <property name="text">
    <string>NETWORK TYPE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="matrixTypeButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       #30D5C8;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:       #FFFFFF;
    border:                 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>MATRIX</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="graphTypeButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       #30D5C8;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:       #FFFFFF;
    border:                 2px solid white;
}</string>
   </property>
   <property name="text">
    <string>GRAPH</string>
   </property>
  </widget>
  <widget class="QLabel" name="hiddenLayersLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;

}</string>
   </property>
   <property name="text">
    <string>HIDDEN LAYERS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="layersBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	background: #202020;
}

QComboBox QAbstractItemView{
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	background: #202020;
	selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #30D5C8 , stop:1 #4169E1);;
}

QComboBox::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #30D5C8 , stop:1 #4169E1);
}


</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="trainingLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	border-style: solid;
	border-width: 4px 0px 4px 0px;
	border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #30D5C8 , stop:1 #4169E1);
}</string>
   </property>
   <property name="text">
    <string>TRAINING</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="epochNumberLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;

}</string>
   </property>
   <property name="text">
    <string>EPOCH NUMBER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="epochBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	background: #202020;
}

QComboBox QAbstractItemView{
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	background: #202020;
	selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #30D5C8 , stop:1 #4169E1);;
}

QComboBox::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #30D5C8 , stop:1 #4169E1);
}


</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="crossValidationCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 10pt ;
	font-weight: bold;
	color: #FFFFFF;
	background-color: #202020;
}
</string>
   </property>
   <property name="text">
    <string>CROSS VALIDATION</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="kGroupsSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>255</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	background: #202020;
}</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="testsLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	border-style: solid;
	border-width: 4px 0px 4px 0px;
	border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #30D5C8 , stop:1 #4169E1);
}</string>
   </property>
   <property name="text">
    <string>TESTS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="selectonPartSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>355</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;
	background: #202020;
}</string>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="selectionPartLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt ;
	font-weight: bold;
	color: #FFFFFF;

}</string>
   </property>
   <property name="text">
    <string>SELECTION PART</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kGroupsLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 10pt ;
	font-weight: bold;
	color: #FFFFFF;

}</string>
   </property>
   <property name="text">
    <string>K-GROUPS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
